src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs

```diff
@@ -60,10 +60,10 @@ public KeyInfoX509Data(X509Certificate cert, X509IncludeOption includeOption)
                     // Can't honor the option if we only have a partial chain.
                     if ((chain.ChainStatus.Length > 0) &&
                         ((chain.ChainStatus[0].Status & X509ChainStatusFlags.PartialChain) == X509ChainStatusFlags.PartialChain))
-                        throw new CryptographicException(CAPI.CERT_E_CHAINING);
+                    throw new CryptographicException(unchecked((int)Interop.Crypt32.CertChainPolicyErrors.CERT_E_CHAINING));
```

Comment:
So... the problem with the CAPI constants is that on .NET Framework there's only one crypto stack, so `new CryptographicException(int)` looks up the error message.  On .NET Core there are 2.5 crypto stacks, so error messages don't get resolved.

This should probably change to message-based exception instead of a value-based exception.